doctr/models/recognition/master/pytorch.py

```diff
@@ -213,7 +216,7 @@ def decode(self, encoded: torch.Tensor) -> torch.Tensor:
         b = encoded.size(0)
 
         # Padding symbol + SOS at the beginning
-        ys = torch.zeros((b, self.max_length), dtype=torch.long, device=encoded.device) * (self.vocab_size + 2)  # pad
+        ys = torch.ones((b, self.max_length), dtype=torch.long, device=encoded.device) * (self.vocab_size + 2)  # pad
```

Comment:
Let's avoid the multiplication:
```python
ys = torch.full((b, self.max_length), self.vocab_size + 2, dtype=torch.long, device=encoded.device) # pad
```

Reply:
Let me check this again, if i remember there was a problem with full but i will check it again :)